suggester/src/main/java/org/opengrok/suggest/Suggester.java

```diff
@@ -87,6 +91,12 @@
 
     private final int rebuildParallelismLevel;
 
+    private boolean rebuilding;
```

Comment:
Should be `volatile`.

Reply:
I assume this is done so that it is not optimized away ?